GH Action on Windows failing du to Sphinx

### Problem Description

All Windows GH based on master seemed to be failing:

```
CUSTOMBUILD : warning : Environment variable ECAL_GH_API_KEY not set. Skipping generating download tables. [D:\a\ecal\_build\complete\doc\documentation_sphinx.vcxproj]
  
CUSTOMBUILD : Sphinx version error :  [D:\a\ecal\_build\complete\doc\documentation_sphinx.vcxproj]
  The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'D:\a\ecal\_build\complete\CMakeFiles\93e7ee793253348722d05d5549b0021b\documentation_sphinx.rule;D:\a\ecal\ecal\doc\CMakeLists.txt' exited with code 2. [D:\a\ecal\_build\complete\doc\documentation_sphinx.vcxproj]
```

### How to reproduce

Trigger any eCAL build on GH actions.

### How did you get eCAL?

Other

### Environment

---

### eCAL System Information

_No response_

--- a/doc/requirements.txt
+++ b/doc/requirements.txt
@@ -9,5 +9,8 @@ exhale
 Jinja2
 sphinxcontrib-youtube<=1.2
 sphinxcontrib-apidoc
-sphinxcontrib-applehelp<=1.0.7 # 2024-01-15: applehelp 1.0.8 needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
-sphinxcontrib-devhelp<=1.0.5  # 2024-01-15: devhelp 1.0.6 needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-applehelp<=1.0.7        # 2024-01-15: Newer packages needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-devhelp<=1.0.5          # 2024-01-15: Newer packages need Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-htmlhelp<=2.0.4         # 2024-01-15: Newer packages (probably) need Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-qthelp<=1.0.6           # 2024-01-15: Newer packages (probably) need Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-serializinghtml<=1.1.9  # 2024-01-15: Newer packages (probably) need Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4

--- a/doc/requirements.txt
+++ b/doc/requirements.txt
@@ -9,4 +9,4 @@ exhale
 Jinja2
 sphinxcontrib-youtube<=1.2
 sphinxcontrib-apidoc
-
+sphinxcontrib-applehelp<=1.0.7 # 2024-01-15: applehelp 1.0.8 needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4

--- a/doc/requirements.txt
+++ b/doc/requirements.txt
@@ -10,3 +10,4 @@ Jinja2
 sphinxcontrib-youtube<=1.2
 sphinxcontrib-apidoc
 sphinxcontrib-applehelp<=1.0.7 # 2024-01-15: applehelp 1.0.8 needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4
+sphinxcontrib-devhelp<=1.0.5  # 2024-01-15: devhelp 1.0.6 needs Sphinx 5 or up, but our old sphinx-book-theme 0.3.3 requires Sphinx 4

